exec/cnex/cell.c

```diff
@@ -9,6 +9,7 @@
 
 #include "array.h"
 #include "dictionary.h"
+#include "object.h"
```

Comment:
Are these intended to be in alphabetical order?

Reply:
Uh.  Opps. Yes.  Fixed.